nuxt/index.js

```diff
@@ -0,0 +1,26 @@
+export default function vueSvgInlineLoaderModule(moduleOptions) {
+	const options = Object.assign(
+		{},
+		this.options.vueSvgInlineLoader,
+		moduleOptions
+	);
+
+	this.extendBuild((config) => {
+		const vueRule = config.module.rules.find(
+			({ test }) => test.toString() === /\.vue$/i.toString()
+		);
+		vueRule.use = [
+			{
+				loader: vueRule.loader,
+				options: vueRule.options,
+			},
+			{
+				loader: join(__dirname, "../index.js"),
+				loader: "vue-svg-inline-loader",
+				options,
+			},
+		];
+		delete vueRule.loader;
+		delete vueRule.options;
```

Comment:
I think this part should be refactored for case if there already is [Vue](https://github.com/vuejs/vue-next) rule with `use` key. This example was just dummy [Nuxt](https://github.com/nuxt/nuxt.js) app without any modifications. Real world apps would most probably have some modifications in place and therefore we should account for them if we want to provide universal solution as module (opposed to manually modifying this rule in `nuxt.config.js` file).

Logic should be something like this:
```javascript
if(vueRule.use) vueRule.use.push({ /* vue-svg-inline-loader rule */ });
else { /* current logic */ }
```